tests/headers_test.cc

```diff
@@ -821,10 +821,10 @@ TEST(headers_test, raw_headers_are_case_insensitive)
         step.apply(cursor);
 
         // or the header you try and get, it should work:
-        ASSERT_FALSE(request.headers().tryGetRaw("Custom-Header").isEmpty());
-        ASSERT_FALSE(request.headers().tryGetRaw("CUSTOM-HEADER").isEmpty());
-        ASSERT_FALSE(request.headers().tryGetRaw("custom-header").isEmpty());
-        ASSERT_FALSE(request.headers().tryGetRaw("CuStOm-HeAdEr").isEmpty());
+        ASSERT_FALSE(request.headers().tryGetRaw("Custom-Header").has_value());
```

Comment:
Changing from `isEmpy()` to `has_value()` inverts the boolean returned, so you should also change `ASSERT_FALSE` to `ASSERT_TRUE` ;)